src/tribler/ui/src/pages/Downloads/Trackers.tsx

```diff
@@ -76,6 +76,12 @@ export default function Trackers({ download }: { download: Download }) {
         }
     ]
 
+    if (download.trackers.length === 0)
```

Comment:
Doesn't this mean that you'll never be able to add trackers to a download with no trackers? Maybe the safer option would be to look at `status_code`?